docs/application/get-started/applications.md

```diff
@@ -4,9 +4,9 @@ Tizen is a user-interactive and service-oriented open source project that allows
 
 ## Tizen Profiles
 
-Tizen is built to work on a wide variety of platforms with a focus on embedded devices. In order to accommodate the various types of devices, a set of profiles has been defined to make it easier to develop applications for specific purposes and device types. Since Tizen 3.0, there are 3 profile types you can choose from: mobile, wearable, and TV. The mobile profile is designed for smart phones, the wearable profile is designed for smart watches, and the TV profile is designed for smart TVs.
+Tizen is built to work on a wide variety of platforms with a focus on embedded devices. To accommodate various types of devices, a set of profiles has been defined to make it easier to develop applications for specific purposes and different device types. Tizen supports four profile types, which are mobile, wearable, TV and IoT. The mobile profile is designed for smartphones, the wearable profile is designed for smartwatches, the TV profile is designed for smart TVs, and the IoT profile is designed for IoT-based smart devices.
```

Comment:
@bson1012 
The current rephrase by you is as follows:

Tizen is built to work on a wide variety of platforms with a focus on embedded devices. To accommodate various types of devices, a set of profiles has been defined to make it easier to develop applications for specific purposes and different device types. Tizen supports four profile types, which are mobile, wearable, TV and IoT. The mobile profile is designed for smartphones, the wearable profile is designed for smartwatches, the TV profile is designed for smart TVs, and the IoT profile is designed for IoT-based smart devices.

You have missed a serial comma after TV 

Tizen supports four profile types, which are mobile, wearable, TV, and IoT. 

Kindly add a comma. 

For your information: 
We use **such as** to introduce an example or examples of something we mention. We normally use a comma before such as when we present a list of examples. Where there is just one example, we don’t need a comma. So, there is no question of the list being complete or not. 

(https://dictionary.cambridge.org/grammar/british-grammar/such-as)

Line 9: 
Also, kindly add a comma after Web[Serial comma rule]https://www.grammar.com/serial-comma-rule)
Both mobile and wearable profiles are supported in native, Web, and .NET application types. The TV profile is supported in Web and .NET application types, whereas the IoT profile is supported in native and .NET application types.

Thanks!